### PR TITLE
Files to be code reviewed

### PR DESCRIPTION
**Pull request by johntopleyons**

### CODE REVIEW

File: models/classifiertypeselectors.go [hunk 1]
```diff
@@ -1,110 +0,0 @@
...
-	// We need to run two queries first to check if the survey and classifier type selector both exist
-	// so an HTTP 404 can be correctly returned if the don't exist. Without this check and HTTP 204 is
-	// incorrectly return for an invalid survey ID or classifier type selector ID.
-	err := getSurveyID(surveyID)
-	if err != nil {
-		return nil, err
-	}
-
-	err = getClassifierTypeSelectorID(classifierTypeSelectorID)
-	if err != nil {
```
Comment (necrophonic) [inline models/classifiertypeselectors.go]: Dunno if it's just github? But there seems to be some odd indentation and formatting - are you running `gofmt`?

File: models/classifiertypeselectors.go [hunk 2]
```diff
@@ -1,110 +0,0 @@
...
-func GetClassifierTypeSelector(surveyID string, classifierTypeSelectorID string) (*ClassifierTypeSelector, error) {
-	// We need to run two queries first to check if the survey and classifier type selector both exist
-	// so an HTTP 404 can be correctly returned if the don't exist. Without this check and HTTP 204 is
-	// incorrectly return for an invalid survey ID or classifier type selector ID.
-	err := getSurveyID(surveyID)
-	if err != nil {
-		return nil, err
-	}
-
-	err = getClassifierTypeSelectorID(classifierTypeSelectorID)
```
Comment (necrophonic) [inline models/classifiertypeselectors.go]: Just returning an ``err`` so could just do as an inline ``if``

File: models/surveys.go [hunk 3]
```diff
@@ -1,86 +0,0 @@
...
-		return nil, err
-	}
-
-	return survey, nil
-}
-
-func getSurveyID(surveyID string) error {
-	var id string
-	err := db.QueryRow("SELECT id FROM survey.survey WHERE id = $1", surveyID).Scan(&id)
-	if err != nil {
```
Comment (necrophonic) [inline models/surveys.go]: Could just return ``err`` rather than needing to check it and then return it or ``nil`` anyway

File: surveysvc.go [hunk 4]
```diff
@@ -1,192 +0,0 @@
...
-	if err == nil {
-		ps := appEnv.Port
-		port = ":" + strconv.FormatInt(int64(ps), 10)
-		postgresServer, err := appEnv.Services.WithTag("postgresql")
-		logInfo("Found Cloud Foundry environment")
-
-		if err == nil {
-			uri, found := postgresServer[0].CredentialString("uri")
-
-			if found {
```
Comment (necrophonic) [inline surveysvc.go]: What if not found?

File: surveysvc.go [hunk 5]
```diff
@@ -1,192 +0,0 @@
...
-}
-
-func main() {
-	port := ":8080"
-	dataSource := "postgres://postgres:password@localhost/postgres?sslmode=disable"
-	appEnv, err := cfenv.Current()
-
-	if err == nil {
-		ps := appEnv.Port
-		port = ":" + strconv.FormatInt(int64(ps), 10)
```
Comment (necrophonic) [inline surveysvc.go]: Does the env really come in as an int?

File: models/classifiertypeselectors.go (hunk 2 above)
Comment (johntopleyons) [inline models/classifiertypeselectors.go]: I'm not quite sure what you mean, can you show me the code please?

File: models/classifiertypeselectors.go (hunk 1 above)
Comment (johntopleyons) [inline models/classifiertypeselectors.go]: Yes, I've got VS Code configured to run it upon save. It looks OK to me...? https://github.com/ONSdigital/rm-survey-service/blob/master/models/classifiertypeselectors.go

File: models/surveys.go (hunk 3 above)
Comment (johntopleyons) [inline models/surveys.go]: Good catch, thanks.

File: surveysvc.go (hunk 5 above)
Comment (johntopleyons) [inline surveysvc.go]: Yes: https://github.com/cloudfoundry-community/go-cfenv/blob/3b3d4d0e3aaeae8f0b7d0d41257228e92cef463e/application.go#L11

File: surveysvc.go (hunk 4 above)
Comment (johntopleyons) [inline surveysvc.go]: Then it's bad news :-( Rob S actually added that code, I'm blaming him! I'll add an `else` with some logging.

**Pull request closed without merging**